yarn-project/end-to-end/src/e2e_crowdfunding_and_claim.test.ts

```diff
@@ -303,34 +292,10 @@ describe('e2e_crowdfunding_and_claim', () => {
       .send({ authWitnesses: [witness] })
       .wait();
 
-    // Calling the function normally will fail as msg_sender != operator
+    // The following should fail as msg_sender != operator
     await expect(
       crowdfundingContract.withWallet(donorWallets[1]).methods.withdraw(donationAmount).send().wait(),
     ).rejects.toThrow('Assertion failed: Not an operator');
-
-    // Instead, we construct a call and impersonate operator by skipping the usual account contract entrypoint...
```

Comment:
The following was unrelated to testing the `Crowdfunding` contract and instead it was introduced in [this PR](https://github.com/AztecProtocol/aztec-packages/pull/7404) to test that an exploit no longer works.

I am unable to mock here oracles since it's an e2e test and I would need to mock the tagging sender oracles here for the test to continue working. I feel like we no longer need this so I decided to just drop it from here.

@MirandaWood do you think dropping this is fine?

Reply:
Yep, dropping that should be fine! Feel free to take this with a pinch of salt as I worked on that PR a year ago, but we know that exploit is covered, so should be ok